modules/openshift-cluster-limits.adoc

```diff
@@ -13,71 +13,72 @@
 | 2,000
 | 2,000
 | 2,000
-|TBD
+| 250
```

Comment:
Is it really 250? Not 2500?

Reply:
@warmchang we lowered the value to 250 as it's the maximum node scale we tested for OCP 4.1. We can bump it up to 2000 if and when we get an opportunity to run tests at a higher node count.

Reply:
@chaitanyaenr Got it, thank you for reply!